server/pyproject.toml

```diff
@@ -5,3 +5,11 @@ asyncio_mode = "auto"
 markers = [
     "app_validate_responses",
 ]
+
+[tool.isort]
+case_sensitive = false
+order_by_type = false
+force_sort_within_sections = true
+multi_line_output = 2
+line_length = 100
+combine_as_imports = true
```

Comment:
Can you please migrate this file to `setup.cfg`, we are already using it and I want to avoid the package config bloat :pray:

Reply:
Or migrate `setup.cfg` to `pyproject.toml`, whatever.